index.js

```diff
@@ -116,26 +123,57 @@ var cleanURL = function(address) {
 // Returns nothing.
 var clone = function(address, rootPath, callback) {
   var url = cleanURL(address);
-  logger.log('Cloning git repo "' + url + '" to "' + rootPath + '"...');
-  tmp.dir({unsafeCleanup: true}, function(error, tempPath, cleanupCallback) {
-    if (error != null) {
-      return callback(new Error("Temp dir error: " + error.toString()));
-    }
-    var cmd = 'git clone ' + url + ' ' + tempPath;
-    exec(cmd, function(error, stdout, stderr) {
-      if (error != null) {
-        return callback(new Error("Git clone error: " + stderr.toString()));
-      }
-      rimraf(sysPath.join(tempPath, '.git'), function(error) {
+  var cacheDir = sysPath.join(os.homedir(), '.init-skeleton', 'cache');
+  var repoHash = sha1Digest(url);
```

Comment:
Do we? I mean, we're literally cloning the whole repo and if the cached one exists, we do `git pull`